h2o-automl/src/test/java/hex/ModelBuilderAutoMLTest.java

```diff
@@ -0,0 +1,142 @@
+package hex;
+
+import ai.h2o.automl.Algo;
+import hex.tree.SharedTreeModel;
+import hex.tree.gbm.GBMModel;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import water.*;
+import water.fvec.Frame;
+
+import java.util.Random;
+
+import static org.junit.Assert.assertEquals;
+
+public class ModelBuilderAutoMLTest extends TestUtil {
```

Comment:
I'm testing copy of MB based on the ability to be trained and score same results on test split with arbitrary algo. This is the reason why I could not place test in the core module's tests as I needed algos. I assumed that testing with just arbitrary algo is enough as the main goal is to test ability to create new training job and use copy of the original data. Maybe adding test to `h2o-core` module with just  checking of the data being copied also makes sense. Will rely on reviewers comments.